### DISCUSSION TITLE
Deconz RaspBee II - Zigbee2MQTT fails to start - zigbee-herdsman (0.14.46)

### DISCUSSION THREAD

**Discussion by cralwalker** [Koenkk/zigbee2mqtt#13774] [Devices]
What logs can I pull to _really_ pinpoint where the issue need to be resolved?

Logs from the HomeAssistant Addon

```
> zigbee2mqtt@1.27.0 start
> node index.js
Zigbee2MQTT:info  2022-08-31 13:42:32: Logging to console and directory: '/config/zigbee2mqtt/log/2022-08-31.13-42-32' filename: log.txt
Zigbee2MQTT:info  2022-08-31 13:42:32: Starting Zigbee2MQTT version 1.27.0 (commit #unknown)
Zigbee2MQTT:info  2022-08-31 13:42:32: Starting zigbee-herdsman (0.14.46)
Zigbee2MQTT:error 2022-08-31 13:42:42: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2022-08-31 13:42:42: Failed to start zigbee
Zigbee2MQTT:error 2022-08-31 13:42:42: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2022-08-31 13:42:42: Exiting...
Zigbee2MQTT:error 2022-08-31 13:42:42: undefined
[13:42:45] INFO: Handing over control to Zigbee2MQTT Core ...

```

**Background:** the original install and configuration went smooth and have been running stable for many months. After moving from microSD card to USB connected SSD by backup/restore, the Zigbee2mqtt has not worked, even after remove/clean install.

**Comment by Koenkk** [comment]
> USB connected SSD by backup/restore, the Zigbee2mqtt has not worked, even after remove/clean install

This is known to cause issues, make sure to connected the SSD via an USB powered hub

**Reply by cralwalker** [reply]
`make sure to connected the SSD via an USB powered hub`
Thank you. It has been done.
 
Herdsman is still bailing out. What other logs can I apply to see where it goes wrong?

```
> zigbee2mqtt@1.27.0 start
> node index.js
Zigbee2MQTT:info  2022-09-01 14:47:40: Logging to console and directory: '/config/zigbee2mqtt/log/2022-09-01.14-47-40' filename: log.txt
Zigbee2MQTT:info  2022-09-01 14:47:40: Starting Zigbee2MQTT version 1.27.0 (commit #unknown)
Zigbee2MQTT:info  2022-09-01 14:47:40: Starting zigbee-herdsman (0.14.46)
Zigbee2MQTT:error 2022-09-01 14:47:50: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2022-09-01 14:47:50: Failed to start zigbee
Zigbee2MQTT:error 2022-09-01 14:47:50: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2022-09-01 14:47:50: Exiting...
Zigbee2MQTT:error 2022-09-01 14:47:50: undefined
[14:47:53] INFO: Handing over control to Zigbee2MQTT Core ...
```

**Reply by cralwalker** [reply]
Somewhat sheepish I just realised migrating from microSD card to USB connected SSD by backup/restore does not restore the root file 'config.txt' edited during the original Deconz installation.

Reference
https://github.com/home-assistant/addons/blob/master/deconz/DOCS.md#using-a-raspbee

**Comment by haegar33** [comment]
I am picking up this old thread as I am trying the transition deCONZ >Z2M but (of course) it fails ..
I got the Raspbee II adapter and the only config parameters applicable for this adapter are set:

> serial:
>    adapter: deconz
>    baudrate: 38400
>    port: /dev/serial1

Note: /dev/serial1 points to /dev/ttyAMA0
Even though I set log level to `DEBUG` I only get the same error message as above:

> Zigbee2MQTT:error 2023-11-18 12:31:08: Error while starting zigbee-herdsman
> Zigbee2MQTT:error 2023-11-18 12:31:08: Failed to start zigbee
> Zigbee2MQTT:error 2023-11-18 12:31:08: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start

I got the deconz service stopped before but no difference
I try to inject the same network channel, networkKey as deconz but no difference.

Are there some more debug logs available to trace the error?
